tests/integration/test_integration.py

```diff
@@ -133,6 +133,29 @@ def test_inin007_meta_tags(dash_duo):
         assert meta_tag.get_attribute("content") == meta_info["content"]
 
 
+def test_inin007b_change_viewport_meta_tag(dash_duo):
```

Comment:
I'd rather we not squeeze test numbers in between with `b` or such - makes it harder to run just one test with eg `pytest -k inin007`. Just shift some of the other test numbers so this one can have its own. Maybe the ones up front since this file starts at `inin004`?